src/main/java/org/openbmp/db_rest/resources/Updates.java

```diff
@@ -320,9 +320,10 @@ public Response getUpdatesOverTime(@QueryParam("searchPeer") String searchPeer,
                 + interval + ") as IntervalTime,\n");
         query.append("               count(*) as Count\n");
         query.append("      FROM path_attr_log l\n");
-		query.append("      JOIN bgp_peers p ON (l.peer_hash_id = p.hash_id)\n");
-		query.append("      JOIN routers r ON (p.router_hash_id = r.hash_id)\n");
-		query.append("      JOIN collectors c ON (r.collector_hash_id = c.hash_id)\n");        query.append("      WHERE l.timestamp >= "+startTimestamp +" AND l.timestamp <= " + endTimestamp + "\n");
+//		query.append("      JOIN bgp_peers p ON (l.peer_hash_id = p.hash_id)\n");
+//		query.append("      JOIN routers r ON (p.router_hash_id = r.hash_id)\n");
+//		query.append("      JOIN collectors c ON (r.collector_hash_id = c.hash_id)\n");
```

Comment:
We can work on adding this back later.... but sometimes nested and STRAIGHT_JOINS are better for this type of query.